yoti/YotiWidget.php

```diff
@@ -26,8 +26,14 @@ public function widget($args, $instance)
     {
         wp_enqueue_style('yoti-asset-css', plugin_dir_url(__FILE__) . 'assets/styles.css');
         $config = YotiHelper::getConfig();
+        $widgetTitleHtml = '';
+        $widgetDefaultTitle = 'New title';
+        // Make sure the title is no empty and is not the default text
+        if(!empty($instance['title']) && $instance['title'] !== $widgetDefaultTitle){
+            $widgetTitleHtml = '<h3 class="yoti-widget-title">' . $instance['title']. '</h3>';
```

Comment:
Why not default style: `<h2 class="widget-title">Authenticate with Yoti</h2>`?

Reply:
I am using `yoti-widget-title` instead of wordPress default class, because the Tittle looks too big imo. So I will be able to customise the display as it fits. But if you think the way WordPress is displaying the title is fine, I am happy to change it.

Reply:
For me, it's the other way around. The Yoti header looks too big. I would prefer keeping the default WordPress styling.

Reply:
Ok cool - we can apply WP style.